server/views/autherror.njk

```diff
@@ -3,10 +3,17 @@
 {% set pageId = 'auth-error' %}
 
 {% block content %}
-  <h1 class="govuk-heading-l">
-    Authorisation Error
-  </h1>
-  <p>
-    You are not authorised to use this application.
-  </p>
+
+  <section class="app-container govuk-body">
+
+    <h1 class="govuk-heading-l">
+      Authorisation Error
+    </h1>
+    <p>
+      You are not authorised to use this application.
+    </p>
+
+  </section>
+
+  {{ contactHelpdeskBanner() }}
```

Comment:
Because of the way the govuk template works there is no obvious hook to enable us to inject content like this - it needs to sit just above the footer, but in respect of the markup it is within the main content.
The best I could do was to write a macro, and add it to every page just below the main `app-container` `section`
(there were a couple of pages like this one that didnt have a `section` to start with, so I've had to add it.

Basically, in respect of the markup, the new "contact the helpdesk" banner is a `section` that sits as a sibling to the main content `section`:

```
<section class="app-container govuk-body">
   ... all the main page content lives in here
</section>

<section class="contact-helpdesk">
    <a href="" class="govuk-link govuk-link--inverse">Contact the helpdesk</a>
</section>
```